src/front/js/component/Testimonials.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import user1 from "../../img/paty.jpg";
+import user2 from "../../img/nan.jpg";
+import "../../styles/Testimonials.css";
+
+const Home = () => {
+  return (
+    <React.Fragment>
+      <section className="content-container">
+        <div className="textArea">
+          <h2>Testimonials</h2>
+          <p>What others parents says about our babbysitters</p>
+        </div>
+        <div className="cust-container">
```

Comment:
Another great place where we could transform all this HTML into a Component with props!
We could call it "Testimonial", what do you think? 😎

Reply:
I like this idea, but remember we have 3 weeks left now and we need to have the Minimum Value Product. We can create a card for this in Trello as an improvement, wdty? :)